site/docs/v1/tech/apis/applications.adoc

```diff
@@ -162,6 +163,125 @@ include::docs/v1/tech/apis/_standard-put-response-codes.adoc[]
+
+[field]#tenantId# [type]#[UUID]# [optional]#Optional#::
+Restricts the results to Applications belonging to the given Tenant. This parameter will be overridden if the request contains an `X-FusionAuth-TenantId` header, or if the supplied API key is scoped to a specific Tenant.
+
+[field]#numberOfResults# [type]#[Integer]# [optional]#Optional# [default]#defaults to `25`#::
+The number of results to return from the search.
+
+[field]#orderBy# [type]#[String]# [optional]#Optional# [default]#defaults to `insertInstant DESC`#::
+The database column to order the search results on plus the order direction.
++
+The columns you can use for this are:
++
+ * `id` - the UUID of the Application
+ * `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Application was created
+ * `name` - the Application name
+ * `tenant` - the name of the Tenant in which the Application belongs
++
+For example, to order the results by the insert instant in a descending order, the value would be provided as `insertInstant DESC`. The final string is optional can be set to `ASC` or `DESC`. If omitted, the default sort order is `ASC`.
+
+[field]#startRow# [type]#[Integer]# [optional]#Optional# [default]#defaults to `0`#::
+The offset row to return results from. If the search has 200 records in it and this is 50, it starts with row 50.
+
+include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-scoped-operation.adoc[]
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Applications using the given search and sort criteria
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/application/search#
+--
+
+When calling the API using a `POST` request you will send the search criteria in a JSON request body.
+
+==== Request Body
+
+[.api]
+[field]#search.name# [type]#[String]# [optional]#Optional#::
+The string to search for in the Application name. This can contain wildcards using the asterisk or percent characters (* or %). This search parameter is case-insensitive.
+
+[field]#search.active# [type]#[Boolean]# [optional]#Optional#::
+Filters on the Application's `active` status. Can be omitted or set to `null` to retrieve both active and inactive Applications.
+
+[field]#search.tenantId# [type]#[UUID]# [optional]#Optional#::
+Filters Applications to those belonging to the specified Tenant. This parameter will be overridden if the request contains an `X-FusionAuth-TenantId` header, or if the supplied API key is scoped to a specific Tenant.
+
+[field]#search.numberOfResults# [type]#[Integer]# [optional]#Optional# [default]#defaults to `25`#::
+The number of results to return from the search.
+
+[field]#search.orderBy# [type]#[String]# [optional]#Optional# [default]#defaults to `insertInstant DESC`#::
+The database column to order the search results on plus the order direction.
```

Comment:
Same comment about reworking this, order suffix, etc. In fact, this is large enough that I would suggest moving it to an include file so that you can share the text between the GET and the POST sections. I think you could probably do everything like that. That makes understanding the doc more complex, but prevents future maintainers from adding a parameter or modifying a desc to the `GET` section and not the `POST` section.

Here's an example: https://raw.githubusercontent.com/FusionAuth/fusionauth-site/master/site/docs/v1/tech/apis/_user-search-request-parameters.adoc

Reply:
Perfect! Thank you for the example. Mark and I talked about this. I think we will use this pattern to repeat the request parameters for `GET` and `POST`.